### PR TITLE
Encryptor Pro (Independent Publisher)

### PR DESCRIPTION
**Pull request by NatiTurts**
- [x] I attest that the connector doesn't exist on the Power Platform today. I've verified by checking the pull requests in GitHub and by searching for the connector on the platform or in the documentation. 
- [x] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [ ] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [x] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [x] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [x] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md. 

Encrypt
Decrypt
Test Operation

### CODE REVIEW

**Comment by NatiTurts** [comment]
> @NatiTurts please read the following Contributor License Agreement(CLA). If you agree with the CLA, please reply with the following information.
> 
> ```
> @microsoft-github-policy-service agree [company="{your company}"]
> ```
> 
> > Options:
> > 
> > * (default - no company specified) I have sole ownership of intellectual property rights to my Submissions and I am not making Submissions in the course of work for my employer.
> > 
> > ```
> > @microsoft-github-policy-service agree
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > 
> > * (when company given) I am making Submissions in the course of work for my employer (or my employer has intellectual property rights in my Submissions by contract or applicable law). I have permission from my employer to make Submissions and enter into this Agreement on behalf of my employer. By signing below, the defined term “You” includes me and my employer.
> > 
> > ```
> > @microsoft-github-policy-service agree company="Microsoft"
> > ```
> 
> Contributor License Agreement

@microsoft-github-policy-service agree

**Comment by NatiTurts** [comment]
@microsoft-github-policy-service agree

File: independent-publisher-connectors/EncryptorPro/README.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+## Prerequisites
+This connector does not require any API subscription or service to work.
+
+## Setting up the connector
+On initial use, you will need to initiate a new connection reference. This will allow for the connector to be used without any API keys or account IDs
+
+## Supported Operations
+The connector supports the following operations:
+* `Encrypt Data`:  Encrypts provided data into AES-256 encryption. 
+* `Dencrypt Data`: Decrypts encrypted data (encrypted using this connector) back into plain text.
```
Comment (vmanoharas) [inline independent-publisher-connectors/EncryptorPro/README.md]: Could you please correct the typo "Dencrypt"?

File: independent-publisher-connectors/EncryptorPro/README.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+## Screenshots
+
+
+
+### For support see [support@turtledovecloudsolutions.com](mailto:support@turtledovecloudsolutions.cmo)
+
+### This connector is provided for use by the community to leverage the data ecnryption within a Microsoft Power Platform ecosystem.
```
Comment (vmanoharas) [inline independent-publisher-connectors/EncryptorPro/README.md]: Could you please correct this typo "ecnryption"?

**Review by vmanoharas — COMMENTED** [review]
Hello @NatiTurts, 

Could you please resolve review comments and address the below swagger validation issues as well?

##[error]The 'default' response should not have schema definition. Schemas should be defined on expected responses only. ValuePath: /paths/~1decrypt~1/post/responses/default
##[error]The 'default' response should not have schema definition. Schemas should be defined on expected responses only. ValuePath: /paths/~1encrypt~1/post/responses/default
##[error]The connector category is invalid. Available categories are: [AI, Business Management, Business Intelligence, Collaboration, Commerce, Communication, Content and Files, Finance, Data, Human Resources, Internet of Things, IT Operations, Lifestyle and Entertainment, Marketing, Productivity, Sales and CRM, Security, Social Media, Website]. ValuePath: /x-ms-connector-metadata/2/propertyValue
##[error]The maximum allowed number of categories for the connector is '2'. ValuePath: /x-ms-connector-metadata/2/propertyValue

File: independent-publisher-connectors/EncryptorPro/README.md (hunk 1 above)
Comment (NatiTurts) [reply independent-publisher-connectors/EncryptorPro/README.md]: Hi, this has been amended.

File: independent-publisher-connectors/EncryptorPro/README.md (hunk 2 above)
Comment (NatiTurts) [reply independent-publisher-connectors/EncryptorPro/README.md]: Hi, this has been amended.

**Comment by NatiTurts** [comment]
@vmanoharas, I have attempted to resolve the swagger default response errors as well. 
Please could you kindly proceed with checks and approval. 

Thanks

**Comment by vmanoharas** [comment]
Hello @NatiTurts, 

I am following up on the above comments, can you please update artifacts as per the comments to proceed for further review?

**Pull request closed without merging**